src/components/Notifications/notification.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import { markNotificationAsRead } from "./../../services/notifications-service";
+
+/**
+ * A component to render one notification.
+ */
+const Notification = ({ notificationFromList }) => {
+
+    const [notification, setNotification] = useState(notificationFromList);
+    console.log(notification.id);
+
+    // create a notification message depending on the type of notification
+    var nstring;
```

Comment:
const or let is more standard Javascript for variable declaration; var has become anti-pattern. Could we rename nstring to something more descriptive?

Reply:
Renamed to notifications string. I wanted to use notifications message, but thought that might be confusing regarding the distinction between notifications message type.